code/modules/reagents/reagent_containers/borghydro.dm

```diff
@@ -21,7 +21,12 @@
 	reagent_ids = list("bicaridine", "kelotane", "anti_toxin", "dexalin", "inaprovaline", "tramadol", "spaceacillin", "stoxin")
 
 /obj/item/reagent_containers/borghypo/rescue
-	reagent_ids = list("tricordrazine", "inaprovaline", "tramadol")
+	reagent_ids = list("dexalin", "inaprovaline", "paracetamol", "iron", "quickclot")
+	// dexalin - prevent dying from oxyloss
+	// iron - replenish oxy
```

Comment:
/datum/reagent/metal/iron/affect_ingest(mob/living/carbon/M, alien, effect_multiplier)
	M.add_chemical_effect(CE_BLOODRESTORE, 0.8 * effect_multiplier)
	
	Only works if you drink it not inject

Reply:
now it works in blood.